modules/shapefile/docs/api-reference/dbf-loader.md

```diff
@@ -0,0 +1,53 @@
+# DBFLoader
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/From-v2.3-blue.svg?style=flat-square" alt="From-v2.3" />
+</p>
+
+Shapefile loader: DBF (attributes)
+
+| Loader                | Characteristic                               |
+| --------------------- | -------------------------------------------- |
+| File Extension        | `.dbf`,                                      |
+| File Type             | Binary                                       |
+| File Format           | Shapefiles                                   |
+| Data Format           | [Table](/docs/specifications/category-table) |
+| Supported APIs        | `load`, `parse`, `parseSync`                 |
+| Decoder Type          | Synchronous                                  |
+| Worker Thread Support | Yes                                          |
+
+## Usage
+
+The `DBFLoader` parses feature attributes from the Shapefile format.
+
+```js
+import {DBFLoader} from '@loaders.gl/shapefile';
+import {load} from '@loaders.gl/core';
+
+const options = {
+  dbf: {
+    encoding: 'utf8'
+  }
+};
+const data = await load(url, DBFLoader, options);
+// [{foo: null}, {foo: 'blue'}, {foo: 'green'}];
+```
+
+## Options
+
+- `encoding`: text encoding of DBF file: usually either `utf8`, or `ascii`/`windows-1252`. For Shapefiles, there's often a `.cpg` file designating the encoding used.
+
+## Format Summary
+
+ESRI Shapefiles are a popular file format for storing geospatial vector data.
```

Comment:
Technically this is some version of " dBase database file. The file type was introduced in 1983 with the introduction of dBASE II. " but used by ESRI as a subformat for Shapefile.